pkg/semver/version.go

```diff
@@ -47,3 +66,91 @@ func (sv ScyllaVersion) SupportFeatureUnsafe(featureVersion semver.Version) bool
 func (sv ScyllaVersion) SupportFeatureSafe(featureVersion semver.Version) bool {
 	return !sv.unknown && sv.version.GTE(featureVersion)
 }
+
+func GetImageVersionAndDigest(imageName, version string) (string, string, error) {
+	// If the version is already in <tag>@<digest> format, extract the semantic version
+	if strings.Contains(version, "@") {
+		semVersion, err := computeSemVersion(version)
+		if err != nil {
+			return "", "", err
+		}
+		return semVersion, version, nil
+	}
+
+	imageReference := fmt.Sprintf("scylladb/%s", imageName)
```

Comment:
Lets not hardcode anything. ScyllaCluster spec accepts also repository which might not contain scylladb substring at all.